pages/storage_and_backup/object_storage/s3_asynchronous_replication/guide.en-gb.md

```diff
@@ -0,0 +1,378 @@
+
+However, you can still replicate delete markers by adding the DeleteMarkerReplication element to your replication configuration rule. DeleteMarkerReplication specifies if delete markers should or should not be replicated (when versioning is enabled, a delete operation is performed on an object it does not actually delete the object but it flags it with a delete marker)
+
+```json
+{
+  "Role": "string",
+  "Rules": [
+    {
+      ...
+      "DeleteMarkerReplication": {
+        "Status": "Enabled"|"Disabled"
+      }
+    }
+  ]
+}
+```
+
+### Checking the Replication Status
+
+The replication status can be used to determine the status of an object that is being replicated. When you request a source object (using `GET object`) or source object metadata (using `HEAD object`), OVHcloud Object Storage returns the replication status via the header `x-amz-replication-status`.
+
+> [!warning]
+> The replication status only applies to objects that are eligible for replication
+
+The x-amz-replication-status can have the following values:
+
+| Source object                                          | Replica Object                                      |
+|--------------------------------------------------------------|--------------------------------------------------------------|
+| COMPLETED | REPLICA |
+| FAILED | n/a as the replica doesn't exist |
+| PENDING | n/a as the replica doesn't exist yet|
+
+> [!warning]
+> When you replicate objects to multiple destination buckets, the value of x-amz-replication-status is COMPLETED only when the source object has been successfully replicated to all the destination buckets, otherwise, the header remains at the PENDING value.
+> If one or more destination fail replication, the value of the header becomes FAILED.
+
+#### Examples of replication configuration
+Simple replication between 2 buckets
+```json
+{
+  "Role": "IAM-role-ARN",
+  "Rules": [
+    {
+      "Status": "Enabled",
+      "Destination": {
+        "Bucket": "arn:aws:s3:::destination-bucket"
+      }
+    }
+  ]
+}
+```
+This configuration will replicate all objects (indicated by the empty Filter field) to the bucket destination-bucket.
+
+#### Replication of delete markers
+
+```json
+{
+  "Role": "IAM-role-ARN",
+  "Rules": [
+    {
+      "Status": "Enabled",
+      "Filter" : {
+        "Prefix": "backup",
+        "Tag": {"Key":"important", "Value":"true"}
+      },
+      "Destination": {
+        "Bucket": "arn:aws:s3:::destination-bucket"
+      },
+      "DeleteMarkerReplication": { "Status": "Disabled" },
+    }
+  ]
+}
+```
+
+This configuration will replicate all objects that have the prefix "backup" and the tag "important" set to "true" to the bucket destination-bucket. Additionally, we indicate that deletion operations in the source bucket should also replicated.
+
+#### Replicating source to multiple regions
+```json
+{
+  "Role": "IAM-role-ARN",
```

Comment:
replace "string" in "role" attribute by "arn:aws:iam::<your_project_id>:role/s3-replication"